.github/workflows/industrial_ci_action.yml

```diff
@@ -16,16 +16,18 @@ jobs:
       matrix:
         env:
           - {
-            ROS_DISTRO: iron, ROS_REPO: main, CCOV_UPLOAD: true,
+            ROS_DISTRO: jazzy, ROS_REPO: main, CCOV_UPLOAD: true,
```

Comment:
Wouldn't it make sense to have this special ccov-build running on rolling?

Reply:
That seems fine to me, will change.